conda/recipes/libkvikio/recipe.yaml

```diff
@@ -62,7 +53,7 @@ cache:
         SCCACHE_NO_CACHE: ${{ env.get("SCCACHE_NO_CACHE", default="") }}
         SCCACHE_RECACHE: ${{ env.get("SCCACHE_RECACHE", default="") }}
         SCCACHE_REGION: ${{ env.get("SCCACHE_REGION", default="") }}
-        SCCACHE_S3_KEY_PREFIX: libkvikio/${{ env.get("RAPIDS_CONDA_ARCH") }}/cuda${{ cuda_key_string }}
+        SCCACHE_S3_KEY_PREFIX: libkvikio/${{ env.get("RAPIDS_CONDA_ARCH") }}/cuda${{ cuda_major }}
```

Comment:
The merge conflict only occurred because we change `cuda_key_string` to `cuda_major` on this line

Upstream had other changes to the environment variables, which we just pulled in